osu.Game/Overlays/Settings/Sections/SkinSection.cs

```diff
@@ -91,6 +92,27 @@ protected override void LoadComplete()
                     skins.SelectRandomSkin();
                 }
             });
+
+            skins.PresentSkinsImport += presentSkinsImport;
+        }
+
+        private void presentSkinsImport(IEnumerable<Live<SkinInfo>> importedSkins)
```

Comment:
It is really weird how this is in the... skin section of the settings? All of the other `PresentFoo` flows are set up at `OsuGame` level and this one should be too. It'd presumably get rid of the weird circularity of `PresentSkinsImport`.